src/azure-cli/azure/cli/command_modules/resource/tests/latest/simple_deploy.json

```diff
@@ -33,8 +33,12 @@
   ],
   "outputs": {
     "NewNSG": {
-      "type": "object",
-      "value": "[reference(parameters('name'))]"
+      // comment
+      /** comment **/
+      "type": "object", /** comment **/
+      "value": "[reference(parameters('name'))]" // comment
+      // comment
+      /** comment **/
```

Comment:
how about /** comment line1
comment line2
comment line3
**/

Reply:
Well, this is handled by jsmin(), let me supplement the test of this case~

Reply:
Updated